layouts/partials/site-head.html

```diff
@@ -48,10 +48,44 @@
 {{"<!-- Main Stylesheet -->"|safeHTML}}
 <link rel="stylesheet" href="{{ `output.css` | absURL }}" media="screen">
 
+{{ "<!-- Google Consent -->" | safeHTML }}
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag() {
+    dataLayer.push(arguments);
+  }
+
+  gtag('consent', 'default', {
+    'ad_storage': 'denied',
```

Comment:
All of the consent types may be found [here]( https://developers.google.com/tag-platform/security/concepts/consent-mode#consent-types).

I couldn't find exactly how HubSpot maps `functionality_storage` and `personalization_storage` in their docs and neither was included in the example code. Functionality is an option listed in the cookie banner, so it felt wise to include it in the script.

The best I could find was this mention in a forum posted a couple of years ago: https://tinyurl.com/2435r9nl

Since I couldn't find a specific mapping for personalization, I decided to stick with functionality as that made the most sense. If I get an error message once I'm able to test, then I guess we'll know if this should be updated.

Reply:
Thanks for doing the research on this - I agree with your choice on selecting `denied` for the default setting.